queries.yaml

```diff
@@ -1779,9 +1832,9 @@ post_process:
   # process may also have created small inners.
   - fn: vectordatasource.transform.drop_small_inners
     params:
-      end_zoom: 15
+      end_zoom: 17
```

Comment:
Change to 16?

Reply:
made a change on tilequeue let's see whether that can fix it: tilezen/tilequeue#407